config/env/all.js

```diff
@@ -5,5 +5,5 @@ var keys = rootPath + '/keys.txt';
 module.exports = {
 	root: rootPath,
 	port: process.env.PORT || 3000,
-    db: process.env.MONGOHQ_URL
+    db: process.env.MONGOHQ_URL || 'mongodb://localhost:27017'
```

Comment:
I assigned/specified a port for it to run locally on my computer, am I suppose to take it out when I deploy?